pages/platform/ai/notebook_tuto_marine_mammal_sounds_classification/guide.en-gb.md

```diff
@@ -19,16 +19,23 @@ The purpose of this tutorial is to show how it is possible to train a model in o
 - Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB);
 - An AI Notebooks project created inside a [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account;
 - A user for AI Notebooks
+- Two [containers Object Storage](https://docs.ovh.com/gb/en/storage/pcs/create-container/) to store the data and the model
 - Your own dataset
 
 ## Instructions
 
 ### Uploading your dataset on Public Cloud Storage
 
-If you want to upload it from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), go to the Object Storage section and create a new object container by clicking `Object Storage`{.action} > `Create an object container`{.action}.
+If you want to upload it from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), go to the Object Storage section and [create](https://docs.ovh.com/gb/en/storage/pcs/create-container/) a new object container by clicking `Object Storage`{.action} > `Create an object container`{.action}.
 
 ![image](images/new-object-container.png){.thumbnail}
 
+> [!primary]
+>
+> With the OVHcloud Control Panel you can only upload files (not folders). You can, for instance, upload a zip file to optimize the bandwidth and unzip it later when you access it threw your JupyterLab.
```

Comment:
Please rephrase:
In the OVHcloud Control Panel, you can upload files but not folders. For instance, you can upload a .zip file to optimize the bandwidth, then unzip it later when accessing it through your JupyterLab.